services-geojson/src/test/java/com/mapbox/geojson/utils/PolylineUtilsTest.java

```diff
@@ -1,24 +1,30 @@
 package com.mapbox.geojson.utils;
 
-import com.mapbox.core.constants.Constants;
+import static com.mapbox.geojson.utils.PolylineUtils.decode;
+import static com.mapbox.geojson.utils.PolylineUtils.encode;
+import static com.mapbox.geojson.utils.PolylineUtils.simplify;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
 import com.mapbox.core.TestUtils;
+import com.mapbox.core.constants.Constants;
 import com.mapbox.geojson.Point;
-
-import org.junit.Assert;
 import org.junit.Test;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class PolylineUtilsTest extends TestUtils {
 
-  private static final String TEST_LINE = "_cqeFf~cjVf@p@fA}AtAoB`ArAx@hA`GbIvDiFv@gAh@t@X\\|@z@`@Z\\Xf@Vf@VpA\\tATJ@NBBkC";
+  private static final String TEST_LINE
+    = "_cqeFf~cjVf@p@fA}AtAoB`ArAx@hA`GbIvDiFv@gAh@t@X\\|@z@`@Z\\Xf@Vf@VpA\\tATJ@NBBkC";
 
   @Test
   public void testDecodePath() {
-    List<Point> latLngs = PolylineUtils.decode(TEST_LINE, Constants.PRECISION_5);
+    List<Point> latLngs = decode(TEST_LINE, Constants.PRECISION_5);
 
     int expectedLength = 21;
-    Assert.assertEquals("Wrong length.", expectedLength, latLngs.size());
+    assertEquals("Wrong length.", expectedLength, latLngs.size());
 
     Point lastPoint = latLngs.get(expectedLength - 1);
     expectNearNumber(37.76953, lastPoint.latitude(), 1e-6);
```

Comment:
should we use a Constants.PRECISION_6 ?

Reply:
The `TEST_LINE` was originally written for precision 5 a long, long time ago ;)